From johannes@sipsolutions.net Wed Feb 28 03:38:54 2007
From: Johannes Berg <johannes@sipsolutions.net>
Date: Wed, 28 Feb 2007 12:38:31 +0100
Subject: driver core: export device_rename
To: Greg KH <greg@kroah.com>, linux-wireless <linux-wireless@vger.kernel.org>
Message-ID: <1172662711.3681.51.camel@johannes.berg>


In wireless we'd like to allow renaming of the phy devices we surface in
sysfs. The base wireless code, however, can be built modular and thus we
need device_rename exported.

Signed-off-by: Johannes Berg <johannes@sipsolutions.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/base/core.c
+++ b/drivers/base/core.c
@@ -1065,7 +1065,7 @@ int device_rename(struct device *dev, ch
 
 	return error;
 }
-
+EXPORT_SYMBOL_GPL(device_rename);
 
 static int device_move_class_links(struct device *dev,
 				   struct device *old_parent,
